views/engineering_garage/edit_project.erb

```diff
@@ -0,0 +1,65 @@
+<h1 class="dcf-txt-h3" id="Edit_Project">Edit Project</h1>
```

Comment:
This page needs a back button to get back to the checkout page with the user details pulled up